packages/core-suggest/readme.md

```diff
@@ -224,6 +224,43 @@ Synchronous operation; dynamically populate items based on input value:
 </script>
 ```
 
+#### Events within shadow root
```

Comment:
Not necessarily useful in a readme, just left it here as a more interactive "proof of concept"